client/src/components/edit/Media.js

```diff
@@ -1,37 +1,66 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { Field, reduxForm, formValueSelector } from 'redux-form'
 import { AppBar, IconButton, Avatar } from 'material-ui'
 import './edit.css'
 import { style } from '../../widgets/styles'
 import Photo from 'material-ui/svg-icons/image/add-a-photo'
 import { StyledPaper } from '../../widgets/StyledPaper'
 import { StyledTextField } from '../../widgets/StyledTextField'
+import {showResults} from "./index";
 
 const videoPlayer = require('./VideoPlaceholder.jpg')
 
-export const Media = props => (
-  <StyledPaper>
-    <div className="EditSectionGrid">
-      <div>
-        <AppBar
-          iconElementLeft={
-            <IconButton>
-              <Photo />
-            </IconButton>
-          }
-          iconStyleLeft={style.appBar}
-          title={<span style={style.title}>Media</span>}
-        />
-      </div>
-      <div className="EditMediaGrid">
-        <div>
-          <StyledTextField text="Video URL" />
-        </div>
-        <div>
-          <div className="EditVideo">
-            <img className="EditImage" src={videoPlayer} />
+class Media extends Component {
+
+  componentDidMount(){
+    const {fetchProfessional} = this.props
+    fetchProfessional(3)
+    this.props.initialize({...this.props})
+  }
+
+  render() {
+    const {handleSubmit} = this.props
+    const videoImage = this.props.profileVideo ? 'EditImageWithVideo' : 'EditImageNoVideo'
+    return (
+      <StyledPaper>
+        <div className="EditSectionGrid">
+          <div>
+            <AppBar
+              iconElementLeft={
+                <IconButton>
+                  <Photo />
+                </IconButton>
+              }
+              iconStyleLeft={style.appBar}
+              title={<span style={style.title}>Media</span>}
+            />
           </div>
+          <form onSubmit={handleSubmit}>
+            <div className="EditMediaGrid">
+              <div>
+                <Field
+                  name="profileVideo"
+                  text="Video URL"
+                  component={StyledTextField}
+                  multiLine
+                />
+              </div>
+              <div>
+                <div className="EditVideo">
+                  <img className={videoImage} src={videoPlayer} />
```

Comment:
If they have a video, will this be replaced with the embedded video instead of an image?